sources/packages/api/src/route-controllers/institution-locations/institution-location.institutions.controller.ts

```diff
@@ -1,107 +1,116 @@
 import {
+  BadRequestException,
+  Body,
   Controller,
   Get,
-  Post,
-  Patch,
-  Param,
-  Body,
-  UnprocessableEntityException,
-  ForbiddenException,
   NotFoundException,
+  Param,
+  Patch,
+  Post,
 } from "@nestjs/common";
-import BaseController from "../BaseController";
 import {
-  InstitutionService,
-  ApplicationService,
-  InstitutionLocationService,
-  FormService,
-} from "../../services";
-import {
-  InstitutionLocationsDetailsDto,
-  InstitutionLocationTypeDto,
-} from "./models/institution-location.dto";
-import { UserToken } from "../../auth/decorators/userToken.decorator";
-import {
-  IInstitutionUserToken,
-  IUserToken,
-} from "../../auth/userToken.interface";
+  ApiBadRequestResponse,
+  ApiNotFoundResponse,
+  ApiTags,
+} from "@nestjs/swagger";
+import { AuthorizedParties } from "../../auth/authorized-parties.enum";
 import {
+  AllowAuthorizedParty,
   HasLocationAccess,
   IsInstitutionAdmin,
-  AllowAuthorizedParty,
+  UserToken,
 } from "../../auth/decorators";
-import { AuthorizedParties } from "../../auth/authorized-parties.enum";
-import { OptionItem } from "../../types";
+import { IInstitutionUserToken } from "../../auth/userToken.interface";
 import {
-  ActiveApplicationSummaryDTO,
-  ActiveApplicationDataDto,
-} from "../application/models/application.model";
+  ApplicationService,
+  FormService,
+  InstitutionLocationService,
+} from "../../services";
+import { ClientTypeBaseRoute } from "../../types";
 import {
-  getUserFullName,
   dateString,
   getISODateOnlyString,
+  getUserFullName,
 } from "../../utilities";
-import { InstitutionLocation, Application } from "../../database/entities";
-import { ApiOkResponse, ApiTags } from "@nestjs/swagger";
-import { InstitutionLocationsControllerService } from "./institution-locations.controller.service";
+import {
+  ActiveApplicationDataAPIOutDTO,
+  ActiveApplicationSummaryAPIOutDTO,
+} from "../application/models/application.dto";
+import BaseController from "../BaseController";
+import { PrimaryIdentifierDTO } from "../models/primary.identifier.dto";
+import { InstitutionLocationControllerService } from "./institution-location.controller.service";
+import {
+  InstitutionLocationAPIOutDTO,
+  InstitutionLocationFormAPIInDTO,
+  InstitutionLocationFormAPIOutDTO,
+} from "./models/institution-location.dto";
 
+/**
+ * Institution location controller for institutions Client.
+ */
+@AllowAuthorizedParty(AuthorizedParties.institution)
 @Controller("institution/location")
-@ApiTags("institution")
-export class InstitutionLocationsController extends BaseController {
+@ApiTags(`${ClientTypeBaseRoute.Institution}-institution`)
```

Comment:
Same point from the previous comments, should it be `@ApiTags(``${ClientTypeBaseRoute.Institution}-institution/location``)`?